bool use_gripper_EE
---
